README.md

```diff
@@ -34,108 +23,102 @@ It's designed to work on infrastructure of the cloud era, and users benefit from
 
-To write and query data, GreptimeDB is compatible with multiple [protocols and clients](https://docs.greptime.com/user-guide/clients/overview).
+Build GreptimeDB binary:
 
-## Resources
+```shell
+make
+```
 
-### Installation
+Run a standalone server:
 
-- [Pre-built Binaries](https://greptime.com/download):
-  For Linux and macOS, you can easily download pre-built binaries including official releases and nightly builds that are ready to use.
-  In most cases, downloading the version without PyO3 is sufficient. However, if you plan to run scripts in CPython (and use Python packages like NumPy and Pandas), you will need to download the version with PyO3 and install a Python with the same version as the Python in the PyO3 version.
-  We recommend using virtualenv for the installation process to manage multiple Python versions.
-- [Docker Images](https://hub.docker.com/r/greptime/greptimedb)(**recommended**): pre-built
-  Docker images, this is the easiest way to try GreptimeDB. By default it runs CPython script with `pyo3_backend` enabled.
-- [`gtctl`](https://github.com/GreptimeTeam/gtctl): the command-line tool for
-  Kubernetes deployment
+```shell
+cargo run -- standalone start
+```
 
-### Documentation
+## Documentation
 
-- GreptimeDB [User Guide](https://docs.greptime.com/user-guide/concepts/overview)
-- GreptimeDB [Developer
-  Guide](https://docs.greptime.com/developer-guide/overview.html)
-- GreptimeDB [internal code document](https://greptimedb.rs)
+- [User guide](https://docs.greptime.com/user-guide/concepts/overview)
+- [API docs](https://greptimedb.rs)
+
+## Extension
 
 ### Dashboard
+
 - [The dashboard UI for GreptimeDB](https://github.com/GreptimeTeam/dashboard)
 
 ### SDK
 
-- [GreptimeDB C++ Client](https://github.com/GreptimeTeam/greptimedb-client-cpp)
-- [GreptimeDB Erlang Client](https://github.com/GreptimeTeam/greptimedb-client-erl)
 - [GreptimeDB Go Ingester](https://github.com/GreptimeTeam/greptimedb-ingester-go)
 - [GreptimeDB Java Ingester](https://github.com/GreptimeTeam/greptimedb-ingester-java)
-- [GreptimeDB Python Client](https://github.com/GreptimeTeam/greptimedb-client-py) (WIP)
-- [GreptimeDB Rust Client](https://github.com/GreptimeTeam/greptimedb-client-rust)
-- [GreptimeDB JavaScript Client](https://github.com/GreptimeTeam/greptime-js-sdk)
+- [GreptimeDB C++ Ingester](https://github.com/GreptimeTeam/greptimedb-ingester-cpp)
+- [GreptimeDB Erlang Ingester](https://github.com/GreptimeTeam/greptimedb-ingester-erl)
+- [GreptimeDB Rust Ingester](https://github.com/GreptimeTeam/greptimedb-ingester-rust)
+- [GreptimeDB JavaScript Ingester](https://github.com/GreptimeTeam/greptime-ingester-js)
 
 ### Grafana Dashboard
 
-Our official Grafana dashboard is available at [grafana](./grafana/README.md) directory.
+Our official Grafana dashboard is available at [grafana](grafana/README.md) directory.
 
 ## Project Status
```

Comment:
We may need to update the project status also.

PTAL @killme2008 @sunng87 @fengjiachun

Reply:
I think its status is going to 1.0 GA.

Reply:
Well. Let me draft an edit in a follow-up.

This PR is large enough :D